helm_drydock/drivers/__init__.py

```diff
@@ -11,11 +11,92 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+from threading import Thread, Lock
+import uuid
+import time
 
+import helm_drydock.objects.fields as hd_fields
+import helm_drydock.statemgmt as statemgmt
+import helm_drydock.objects.task as tasks
+import helm_drydock.error as errors
+
+# This is the interface for the orchestrator to access a driver
+# TODO Need to have each driver spin up a seperate thread to manage
+# driver tasks and feed them via queue
 class ProviderDriver(object):
 
-	__init__(self):
-        pass
-		
+    def __init__(self, orchestrator=None, state_manager=None, **kwargs):
```

Comment:
this also follows proper `docstring` models, and will prove useful if (or rather when) we generate autodocs.